.evergreen/download-mongodb.sh

```diff
@@ -565,16 +565,22 @@ download_and_extract_crypt_shared ()
 {
    MONGO_CRYPT_SHARED_DOWNLOAD_URL=$1
    EXTRACT=$2
+   __CRYPT_SHARED_LIB_PATH=$3
```

Comment:
```suggestion
   __CRYPT_SHARED_LIB_PATH=${3:-CRYPT_SHARED_LIB_PATH}
```

Suggested change is to avoid breaking existing tests. The current Go driver scripts [call with two arguments](https://github.com/mongodb/mongo-go-driver/blob/5073f14299701635e23a2c3c5dd3f14c37f38e03/.evergreen/config.yml#L211.). They will be updated. But in case there are other callers.